stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.2-a
+version: 1.2.0-a
```

Comment:
Sure ok, I was wondering about that. I'll add a note next to the version to make it more obvious.